### ISSUE TITLE
Set default view

### ISSUE THREAD

**Issue by fruitports** [FossifyOrg/Calendar#42]
**Checklist**
 - [x] I made sure that there are no existing issues - open or closed - to which I could contribute my information.
 - [x]  I have read the FAQ and my problem isn't listed.
 - [x]  I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
 - [x]  This issue contains only one feature request.
 - [x]  I have read and understood the [contribution guidelines.](https://github.com/FossifyOrg/General-Discussion?tab=readme-ov-file#faq)

**Is your feature request related to a problem? Please describe.**
Every time I open Calendar from a Kvaesitso calendar widget the view in the app changes to daily. I understand it might be because the intent sent contains the current time, but honestly I don't know much about those things.

**Describe the solution you'd like**
I'd like there to be a way to set a default view that wouldn't change when I open the app from a third party widget.

**Describe alternatives you've considered**
Alternatively, the app could remain in the previous view even when given the time data.

**Additional context**
URI from the interceptor: intent://com.android.calendar/time/1704671957150#Intent;scheme=content;launchFlags=0x3000000;end
After deleting the numbers after /time/ the app doesn't change the view.

**Comment by Aga-C** [comment]
@naveensingh Please check this.

**Comment by gaberilde** [comment]
This drives me insane and needs to be fixed please, you can add a option to disable this at least, then everyone can be happy if they actually want it like this to leave it on but i don't because my daily view is normally empty and it's annoying to have to change it each time.

**Comment by fruitports** [comment]
@gaberilde The alternative I eventually came to was setting up an event in Key Mapper that immediately changes the view to my preferred when I double tap my fingerprint scanner. I imagine you could do something similar in Tasker.

Though, of course, nothing beats the app just working as you wish.

**Comment by gaberilde** [comment]
> @gaberilde The alternative I eventually came to was setting up an event in Key Mapper that immediately changes the view to my preferred when I double tap my fingerprint scanner. I imagine you could do something similar in Tasker.
> 
> Though, of course, nothing beats the app just working as you wish.

Guess i could but trying to avoid accessibility running constantly as much as possible to keep my phone running smoothly but yeah all i want is a option to turn it off would be a lot easier

**Comment by Dan-dilion** [comment]
It's strange that adding an option for this is not being considered, I totally agree with @gaberilde, It drives me crazy, I have to change the view just about every time I open the app!

@Aga-C I wonder if you would reconsider your decision to not add this option. I think it would make the UX a lot better and clearly many users happier myself included.

Thanks again

**Comment by naveensingh** [comment]
I guess this will be revisited since it has been requested again and again, reopening for now.

**Comment by gaberilde** [comment]
Personally i think the view to open from event list widget option should be respected since it does nothing for me.

**Comment by deedend** [comment]
I add myself to the list of people that are annoyed by this missing feature. I use a meteo widget on my home screen, and when I tap in the date it opens the calendar app. I would like to have the option to change the default landing view and keep it like that all the time (in my case, I prefer the monthly and daily view)

**Comment by dsutula13** [comment]
> It's strange that adding an option for this is not being considered, I totally agree with [@gaberilde](https://github.com/gaberilde), It drives me crazy, I have to change the view every time I open the app!
> 
> I wonder how many users turn away from Fossify Calendar because of this configuration?
> 
> [@Aga-C](https://github.com/Aga-C) I wonder if you would reconsider your decision to not add this option. I think it would make the UX a lot better and clearly many users happier, myself included.
> 
> Thanks again

I totally agree. I have replaced all of the system apps with Fossify but I still cannot do it with calendar due to this one behaviour. There is nearly never a time where I would look at the daily view rather than weekly. I already have a widget that tells me the events for the day. The only reason I would want to open the full app is to look at the whole week.

Please consider changing this.

**Comment by C8opmBM** [comment]
The amount of frustration is unquantifiable. Every single time, the empty ugly daily view.
Please, add the option.
Thanks.

**Comment by dharmann** [comment]
I decided to comment just to support the position of who says this is a valuable enhancement. Thank you for your work and effort!

**Comment by gaberilde** [comment]
I do hope this gets added soon, i dont expect it to be too hard and everyone agrees that this view shouldnt be forced me also so the option to disable this should be added.

**Comment by chesio** [comment]
Guys, instead of leaving useless comments here, just add 👍 to the original issue post. Currently this issue is not even in the TOP 25 most-liked issues: https://github.com/FossifyOrg/Calendar/issues/?q=is%3Aissue%20state%3Aopen%20sort%3Areactions-%2B1-desc

As an added benefit, you also annoy less people.

**Comment by ssssamstrup** [comment]
When opening the app with Niagara launcher and Kustom, not from a shortcut/widget, it too defaults to daily as well, rather than 3 day/weekly view which I prefer.
